internal/server/v1beta1/user_test.go

```diff
@@ -31,35 +31,74 @@ func TestGetUserStarredAssets(t *testing.T) {
 		size     = 10
 	)
 	type testCase struct {
+		UserUUID     string
+		UserID       string
 		Description  string
 		ExpectStatus codes.Code
+		WantErr      error
```

Comment:
Rename this to `ValidationErr`. `WantErr` is confusing as it could also mean the err we want to assert with.

Reply:
Also, position it above `ExpectStatus` in the fields and tests.

Reply:
Sure, changing this